src/website/app/demos/ButtonGroup/index.js

```diff
@@ -0,0 +1,25 @@
+/* @flow */
+import examples from './examples';
+import bestPractices from './bestPractices';
```

Comment:
i think you need a

` import { componentTheme } from '../../../../library/ButtonGroup/ButtonGroup';`